client/app.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import $ from 'jquery';
+import MainCarousel from './components/MainCarousel.jsx'
+import './styles/style.css';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      homes: null
+    };
+
+    this.getHomes = this.getHomes.bind(this);
+  }
+
+  //ajax get request
+  getHomes() {
+    var saveHomes = function(data) {
+      this.setState({homes: data});
+    }.bind(this);
```

Comment:
Is there a reason this is a separate function and not right in your success function? @jkwak09